<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="29336" started-at="2022-01-04T15:27:17 IST" finished-at="2022-01-04T15:27:47 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="29336" started-at="2022-01-04T15:27:17 IST" finished-at="2022-01-04T15:27:47 IST">
      <class name="com.orangehrm.testCases.Login_Tc_001">
        <test-method status="PASS" signature="setupSuite()[pri:0, instance:com.orangehrm.testCases.Login_Tc_001@262b2c86]" name="setupSuite" is-config="true" duration-ms="914" started-at="2022-01-04T15:27:16 IST" finished-at="2022-01-04T15:27:17 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupSuite -->
        <test-method status="PASS" signature="setupExtent()[pri:0, instance:com.orangehrm.testCases.Login_Tc_001@262b2c86]" name="setupExtent" is-config="true" duration-ms="166" started-at="2022-01-04T15:27:17 IST" finished-at="2022-01-04T15:27:17 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupExtent -->
        <test-method status="PASS" signature="setUp(java.lang.String)[pri:0, instance:com.orangehrm.testCases.Login_Tc_001@262b2c86]" name="setUp" is-config="true" duration-ms="17729" started-at="2022-01-04T15:27:18 IST" finished-at="2022-01-04T15:27:35 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="loginOrangeHrmTest()[pri:0, instance:com.orangehrm.testCases.Login_Tc_001@262b2c86]" name="loginOrangeHrmTest" duration-ms="8736" started-at="2022-01-04T15:27:35 IST" finished-at="2022-01-04T15:27:44 IST">
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:87)
at org.junit.Assert.assertTrue(Assert.java:42)
at org.junit.Assert.assertTrue(Assert.java:53)
at com.orangehrm.testCases.Login_Tc_001.loginOrangeHrmTest(Login_Tc_001.java:37)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginOrangeHrmTest -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.orangehrm.testCases.Login_Tc_001@262b2c86]" name="tearDown" is-config="true" duration-ms="1199" started-at="2022-01-04T15:27:44 IST" finished-at="2022-01-04T15:27:45 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=loginOrangeHrmTest status=FAILURE method=Login_Tc_001.loginOrangeHrmTest()[pri:0, instance:com.orangehrm.testCases.Login_Tc_001@262b2c86] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="endReport()[pri:0, instance:com.orangehrm.testCases.Login_Tc_001@262b2c86]" name="endReport" is-config="true" duration-ms="1404" started-at="2022-01-04T15:27:45 IST" finished-at="2022-01-04T15:27:47 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endReport -->
      </class> <!-- com.orangehrm.testCases.Login_Tc_001 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
